pkg/scheduler/util/utils.go

```diff
@@ -140,3 +140,227 @@ func MoreImportantPod(pod1, pod2 interface{}) bool {
 	}
 	return GetPodStartTime(pod1.(*v1.Pod)).Before(GetPodStartTime(pod2.(*v1.Pod)))
 }
+
+// Utilities below to build API objects using a "chained" manner
```

Comment:
Shouldn't these be on `pkg/scheduler/testing`? Also, I don't see them being used in this PR.

Reply:
Geez... My bad on rebasing, again. Thanks for the catch @alculquicondor. Updated.